guides/common/modules/con_managing-dhcp-using-smart-proxy.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxy_{context}"]
+= Managing DHCP Using {SmartProxy}
+
+{Project} can integrate with a DHCP service using {SmartProxies}.
+A {SmartProxy} has multiple providers, allowing you to integrate with your existing DHCP infrastructure or deploy a new one.
```

Comment:
`A {SmartProxy} has multiple DHCP service providers that you can use to integrate {Project} with your existing DHCP infrastructure or deploy a new one.` maybe?